cypress/support/utils/product.js

```diff
@@ -96,32 +95,19 @@ export class Product {
   static applyProduct(product) {
     cy.log(`Create new Product ${product.name}`);
     cy.visitWindow('140', 'NEW');
-    cy.writeIntoStringField('Name', product.name);
-
-    cy.selectInListField('M_Product_Category_ID', product.productCategory);
-
-    cy.writeIntoStringField('Description', product.description);
 
-    cy.getCheckboxValue('IsStocked').then(isIsStockedValue => {
-      if (product.isStocked && !isIsStockedValue) {
-        cy.clickOnCheckBox('IsStocked');
-      }
-    });
-    cy.getCheckboxValue('IsPurchased').then(isPurchasedValue => {
-      if (product.isPurchased && !isPurchasedValue) {
-        cy.clickOnCheckBox('IsPurchased');
-      }
-    });
-    cy.getCheckboxValue('IsSold').then(isSoldValue => {
-      if (product.isSold && !isSoldValue) {
-        cy.clickOnCheckBox('IsSold');
-      }
-    });
-    cy.getCheckboxValue('IsDiverse').then(isDiverseValue => {
-      if (product.isDiverse && !isDiverseValue) {
-        cy.clickOnCheckBox('IsDiverse');
-      }
-    });
+    if (product.isStocked) {
```

Comment:
* i think we need to change this pattern
* assume the default is `Y`, but for the test we want `N` and have `product.isStocked == false`
* instead of `if (product.isPurchased)` i think we need to check if `product.isPurchased` is "defined"

Reply:
Done